tools/bcl-test-importer/Makefile

```diff
@@ -0,0 +1,29 @@
+DESTDIR ?= /usr/local/bin/
+CONFIGURATION ?= Debug
+BCL_TEST_IMPORTER_SRC:=BCLTestImporter/ApplicationOptions.cs \
+					BCLTestImporter/BCLTestImporter.csproj \
+					BCLTestImporter/ConsoleWriter.cs \
+					BCLTestImporter/Program.cs \
+					BCLTestImporter/BCLTestProjectGenerator.cs \
+					BCLTestImporter/RegisterTypeGenerator.cs \
+					BCLTestImporter/BCLTestAssemblyDefinition.cs \
+					BCLTestImporter/BCLTestProjectDefinition.cs \
+
+all: BCLTestImporter.exe
+
+BCLTestImporter.exe: $(BCL_TEST_IMPORTER_SRC) Makefile 
+	nuget restore BCLTestImporter/BCLTestImporter.sln
+	msbuild /p:Configuration=$(CONFIGURATION) BCLTestImporter/BCLTestImporter.sln
+
+bcl-test-importer: BCLTestImporter.exe
+	mkbundle --simple ./BCLTestImporter/bin/$(CONFIGURATION)/BCLTestImporter.exe -o $@ -L ./BCLTestImporter/bin/$(CONFIGURATION)
```

Comment:
why does it need to be `mkbundle`d ?

Reply:
I'm doing a cp, so it was simpler, perse, it is not needed :)

Reply:
it's a very costly `cp` replacement ;-) let's not :-)